server.py

```diff
@@ -67,7 +67,7 @@ def receive_signal(signal_number, frame):
         if server not in servers:
             servers[server] = '197001010000'
             
-allowable_methods = ['scan_status:POST', 'send_status:POST', 'sync:GET']
+allowable_methods = ['scan_status:POST', 'send_status:POST', 'sync:GET', 'admin_config:GET', 'admin_status:GET']
```

Comment:
why not keep the protocol in the doc "admin/config" and "admin/status"?

Reply:
I don't really mind - but all the others were xxx_yyy, *AND* wasn't sure how to catch xxx/yyy in this framework - I don't have a strong opinion either way.

Reply:
i was planning on broaching renaming again to "status/scan" and "status/send" too.